generation/new_client/README.md

```diff
@@ -124,55 +124,75 @@ $ sudo apt-get install gh
 $ gh auth login
 ```
 
+## Double check that the library does not already exist within `google-cloud-java`!
+
+Some languages require a new request when a new version of a service is created, but Java manages all versions of the service as a single package, so the client library will automatically be updated to include new versions.
+
 ## Run client generation script
 
 You will run new-client.py script with the following parameters.
-Collect them from the ticket before running the command.
+These parameters will be available in the Cloud Drop link (a YAML file) included in the buganizer request.
+The example in this README uses AlloyDB's [Cloud Drop](https://github.com/googleapis/googleapis/blob/master/google/cloud/alloydb/v1/alloydb_v1.yaml) file as an example.
 
 ### API short name
 
 For convenience of the subsequent commands, define a variable for API short name.
-Get the value from the DevRel Services page (Example: `apikeys`):
+The corresponding value in the Cloud Drop page is `api_short_name`.
+
+Example: `alloydb`
 
 ### Proto path
 
-The script takes "proto path" parameter. This is path from google3's root to the
-directory that contains versions (e.g., "v1" or "v2"). For example, if we
-have 
+The script takes "proto path" parameter. This is path from the internal `google3/third_party/googleapis/stable` root to the
```

Comment:
This doesn't have to be the internal location. It can be "https://github.com/googleapis/googleapis". We then remove the reference to the google3.

Reply:
I clarified this a bit more; let me know what you think. Thanks!